### DISCUSSION TITLE
Qundis Q Caloric heat cost allocator message with only status

### DISCUSSION THREAD

**Discussion by morremeyer** [wmbusmeters/wmbusmeters#830] [Q&A] [answered]
For my Qcaloric 5.5 devices, I currently only get messages like the following:

```
{"media":"heat cost allocation","meter":"qheat","name":"heat_morre","id":"METER_ID","status":"OK","last_year_date":"2022-12-31","device_date_time":"2023-01-29 19:10","device_error_date":"2127-15-31","timestamp":"2023-01-29T18:04:52Z","device":"rtlwmbus[00000001]","rssi_dbm":94}
``` 

Do I interpret this telegram and the test line in https://github.com/weetmuts/wmbusmeters/blob/ed8440c322469f2942e13a8e6594ee8fcd5f3e61/src/driver_qcaloric.cc#L219 correctly as „these telegrams might just have a status and no metering data at all“?

If yes, do we know why that is/might be the case?

**Comment by weetmuts** [comment, marked as answer]
Sure can, some meters send multiple types of telegrams. Some contains meta-data some contain values.
Some contain OMS-compliant meter data, some contain proprietary data (that we have not yet figured out).

So collect some more telegrams to see if you can find a telegram with data.

**Reply by morremeyer** [reply]
Will do, thank you. 

If I find anything interesting that's not yet supported, I'll try opening a PR.

**Reply by PenetratorTest3500** [reply]
Hey,
if you find this like me and you need some input,
search here for Ci-Field, and (as)synchronous bit, acc number:
https://oms-group.org/wp-content/uploads/2024/10/OMS-Spec_Vol2_Primary_v501_01.pdf#page=41

These 3 let you identify different type of tlg.

Have fun reading the sheet :D 

(Annex N holds some Examples
https://oms-group.org/en/specification/
good luck)

**Comment by ilerisoy42** [comment]
Were you able to analyze the qundis heat meters and 5.5 heat share meter devices?
